packages/dds/tree/src/simple-tree/schemaFactory.ts

```diff
@@ -134,13 +134,59 @@ export type ScopedSchemaName<
 > = TScope extends undefined ? `${TName}` : `${TScope}.${TName}`;
 // > = `${TScope extends undefined ? "" : `${TScope}.`}${TName}`;
 
+// TODO:
+// SchemaFactory.array references should link to the correct overloads, however the syntax for this does not seems to work currently for methods unless the they are not qualified with the class.
+// API-Extractor requires such links to be qualified with the class, so it can't work.
+// Since linking the overload set as a whole also doesn't work, these have been made non-links for now.
 /**
- * Builds schema libraries, and the schema within them.
+ * Creates various types of {@link TreeNodeSchema|schema} for {@link TreeNode}s.
  *
  * @typeParam TScope - Scope added as a prefix to the name of every schema produced by this factory.
  * @typeParam TName - Type of names used to identify each schema produced in this factory.
  * Typically this is just `string` but it is also possible to use `string` or `number` based enums if you prefer to identify your types that way.
  *
+ * @remarks
```

Comment:
The alternative declaration syntax that now means POJO mode was something I did not intend anyone to use, but Daniel thought was supported (and the docs failed to be specific enough to imply otherwise) with the semantics now documented here, I'm pretty sure there is no other documentation covering it unless Daniel wrote it, and I'm pretty sure he didn't update any of the existing docs to cover it.

Personally I'm in favor of not supporting POJO mode, but if we really do have a use-case where faking the prototype of the node is desired, we can make that an orthogonal feature not tied to declaration syntax.

Thus I think we should iterate on the API and our internal opinions about it a bit before we write more public facing guidance since I don't think we have internal consensus on what these APIs should be of when/if they should be used.